src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewService.kt

```diff
@@ -269,6 +269,34 @@ class PrisonerIepLevelReviewService(
       }
     }
 
+  @Transactional
+  suspend fun processPrisonerUpdatedEvent(prisonOffenderEvent: HMPPSDomainEvent) {
+    val alertsChanged: Boolean = prisonOffenderEvent.additionalInformation.categoriesChanged?.contains("ALERTS") ?: false
+
+    if (alertsChanged) {
+      updateNextReviewDate(prisonOffenderEvent)
+    } else {
+      log.debug("Ignoring 'prisoner-offender-search.prisoner.updated' event, alerts didn't change: categoriesChanged = ${prisonOffenderEvent.additionalInformation.categoriesChanged}")
```

Comment:
Could be worth also including `additionalInformation.nomsNumber` in the log statement?

Reply:
I'll add 👍